cylc/flow/network/resolvers.py

```diff
@@ -792,16 +794,15 @@ def broadcast(
             # applicable:
             for i, dict_ in enumerate(settings):
                 settings[i] = runtime_schema_to_cfg(dict_)
-
-        if mode == 'put_broadcast':
+        if mode.value == 'put_broadcast':
             return self.schd.task_events_mgr.broadcast_mgr.put_broadcast(
                 cycle_points, namespaces, settings)
-        if mode == 'clear_broadcast':
+        if mode.value == 'clear_broadcast':
             return self.schd.task_events_mgr.broadcast_mgr.clear_broadcast(
                 point_strings=cycle_points,
                 namespaces=namespaces,
                 cancel_settings=settings)
-        if mode == 'expire_broadcast':
+        if mode.value == 'expire_broadcast':
```

Comment:
Curiously, this appears to be back-compatible, I have no idea how?!

Reply:
The Enums are now handled better:
https://github.com/graphql-python/graphene/wiki/v3-release-notes#better-enum-support
https://github.com/graphql-python/graphene/releases/tag/v3.1.1

Reply:
[everyone] Please test backward/forward compatibility of all mutations that use Enums.

* Start a workflow with 8.4.x
* Start a workflow with 8.5.0.dev (this branch)

Then:

* Use the 8.5.0.dev client to issue a mutation to the 8.4.x workflow.
* Use the 8.4.x client to issue a mutation to the 8.5.0.dev workflow.